conans/test/integration/install_selected_packages_test.py

```diff
@@ -19,50 +19,50 @@ def setUp(self):
 
     def install_all_test(self):
         # Should retrieve the three packages
-        self.new_client.run("install Hello0/0.1@lasote/stable --all")
+        self.new_client.run("download Hello0/0.1@lasote/stable")
         p1 = os.path.join(self.new_client.paths.packages(self.ref))
         packages = os.listdir(p1)
         self.assertEquals(len(packages), 3)
 
     def install_some_reference_test(self):
         # Should retrieve the specified packages
-        self.new_client.run("install Hello0/0.1@lasote/stable -p %s" % self.package_ids[0])
+        self.new_client.run("download Hello0/0.1@lasote/stable -p %s" % self.package_ids[0])
```

Comment:
I am thinking, when I see this, that probably it is easier and more easy to automate:

```bash
$ download Hello0/0.1@lasote/stable:shaXXXXXXXXXX
```

And if multiple packages IDs are to be downloaded, then several commands, instead of ``-p`` with Extender.

Just an idea, maybe not useful.

Reply:
Mhh, but you can also split in several commands with `-p`

Reply:
Not about the multiple lines, but because of the fact that you can copy & paste the full packageID, for example, from the conan output after an install. 
But, it is fine as-is too, just personal opinion, I understand that with a ``-p`` argument is more structured